src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -26,6 +26,8 @@ const {
   TaskPriority,
 } = require('ReactPriorityLevel');
 
+const { MAX_SAFE_INTEGER } = Number;
```

Comment:
MAX_SAFE_INTEGER will also trigger the float type in the VM because it's higher than 32 or 31 bit representations which ever is the one used.